telemetry/specs/INT.mdk

```diff
@@ -1165,7 +1139,7 @@ INT Shim Header for TCP/UDP, INT type is hop-by-hop:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |     Type=1    |   Reserved    |  Length = 8   |   DSCP    |R R|
+   |     Type=1    |   Reserved    |  Length = 7   |   DSCP    |R R|
```

Comment:
7 -> 9 ?

(The shim header format is changed at the other PR: https://github.com/p4lang/p4-applications/pull/63 )